arangod/Cluster/Maintenance.cpp

```diff
@@ -320,7 +320,6 @@ void handlePlanShard(StorageEngine& engine, uint64_t planIndex, VPackSlice const
             {DATABASE, dbname},
             {COLLECTION, colname},
             {SHARD, shname},
-            {THE_LEADER, std::string()},
```

Comment:
It turned out that this was no longer needed. Cleanup done as an aside. See TakeoverShardLeadership action.

Reply:
It was always empty anyway.